tacker/sfc/plugin.py

```diff
@@ -196,7 +196,9 @@ def update_sfc(self, context, sfc_id, sfc):
 
 class NeutronClient:
     def __init__(self):
-        auth_url = cfg.CONF.keystone_authtoken.auth_uri + '/v2.0'
+        auth_url = cfg.CONF.keystone_authtoken.auth_uri
+        if re.match('.+(v3/?|v2\.0/?)$', auth_url) is not None:
```

Comment:
Wrong logic this must be `if re.match('.+(v3/?|v2\.0/?)$', auth_url) is None:` otherwise `/v2.0` gets appended if there is a `v2.0` or `v3` included.